app/config/config.yml

```diff
@@ -0,0 +1,59 @@
+imports:
+    - { resource: parameters.yml }
+    - { resource: security.yml }
+    - { resource: services.yml }
+
+# Put parameters here that don't need to change on each machine where the app is deployed
+# http://symfony.com/doc/current/best_practices/configuration.html#application-related-configuration
```

Comment:
Not sure what you mean. This is the standard file after creating the project. I've put some default variables to parameters.yml, but I'm not using symfonys doctrine service anyway, lotgd\core manages the database for us. Real database connection parameters are in .env (which I need to read out manually on windows).